docs/examples/index.md

```diff
@@ -13,3 +13,4 @@ like to see here, please open
 - [Writing tests](./examples/writing-tests)
 - [Changing the source directory](./examples/changing-the-src-dir)
 - [Initializing the server](./examples/init-the-server)
+- [Using nonstandard Fresh](./examples/using-nonstandard-fresh)
```

Comment:
"nonstandard" sounds a little weird to me. What if we rephrase this document as "Fresh canary version" instead? That's likely the most popular reason to go down this route anyway. I prefer this over "nonstandard" as that kinda conveys that APIs we provide aren't good enough to hook into. If such a case pops up I'd rather address any shortcomings in our API instead.

Reply:
I specifically wanted help with this, so I'm happy to change it. But what exactly are you suggesting? The document should be titled `Using Fresh canary version`? And then the following:
* ... a nonstandard version of Fresh. -> ... a canary version of Fresh.
* Nonstandard Fresh in `deno.json` -> Canary Fresh in `deno.json`

Reply:
Yup, those wording changes sound much better imo 👍